test/functional/mempool_datacarrier.py

```
@@ -22,16 +22,18 @@
 
 class DataCarrierTest(BitcoinTestFramework):
     def set_test_params(self):
-        self.num_nodes = 3
+        self.num_nodes = 4
         self.extra_args = [
             [],
             ["-datacarrier=0"],
-            ["-datacarrier=1", f"-datacarriersize={MAX_OP_RETURN_RELAY - 1}"]
+            ["-datacarrier=1", f"-datacarriersize={MAX_OP_RETURN_RELAY - 1}"],
+            ["-datacarrier=1", f"-datacarriersize=2"],
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)